tests/opentracer/test_dd_compatibility.py

```diff
@@ -0,0 +1,156 @@
+import opentracing
+import unittest
+
+import ddtrace
+from tests.opentracer.utils import opentracer_init
+
+
+class TestTracerCompatibility(unittest.TestCase):
+    """Ensure that our opentracer produces results in the underlying ddtracer."""
+
+    def setUp(self):
+        self.ot_tracer, self.dd_tracer = opentracer_init(set_global=False)
```

Comment:
use fixtures instead of a `setUp()` method, unless you need a complex initialization (such as asyncio or Tornado). Also let's avoid `set_global=False`, but provide two fixtures, one that creates the tracer for OpenTracing, the other that also overrides the global tracer without using a `set_global_tracer()`-like function, but overriding it manually in the fixture body.